modules/module_05/homework.md

```diff
@@ -0,0 +1,136 @@
+# Module 05 Homework
```

Comment:
Thanks for merging this into the PR. I'll make sure to split the homework code out into separate files.